README.md

```diff
@@ -44,9 +44,9 @@ Create a JSON object with required data, then use it in such cases:
   "token": "YOUR_TOKEN",
   // For embedded application only. Events also should be dispatched or listened in the root application.
```

Comment:
These texts appear in a dark brown color in readme. Is there a way to show it nicer?